src/AudioAnalysisTools/Harmonics/HarmonicParameters.cs

```diff
@@ -53,6 +62,7 @@ public class HarmonicParameters : CommonParameters
                                 spectrogram,
                                 hp.MinHertz.Value,
                                 hp.MaxHertz.Value,
+                                hp.SmoothingWindow.Value,
```

Comment:
could be null, don't do `.Value` rather pass the nullable into the function and use `is null` as your test